rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1348,6 +1348,12 @@ JobIdData reSubmit(@HeaderParam("sessionid") String sessionId, @PathParam("jobid
             @PathParam("path") PathSegment pathSegment, @Context UriInfo contextInfos)
             throws IOException, RestException;
 
+    @POST
+    @Path("jobs/resubmit")
+    @Produces("application/json")
+    List<JobIdData> reSubmitAll(@HeaderParam("sessionid") String sessionId, @QueryParam("jobids") List<String> jobsId)
```

Comment:
jobsId => jobIds

Reply:
ok

Reply:
@fviale @mboussaa  jobsid is a mistake in many other endpoints as well. It would be nice to fix it elsewhere some day ...

Reply:
its juste the name of the java parameter, not the endpoint parameter, I agree it is a typo

Reply:
actually, no I will revert back to jobsid as it is the syntax used in other endpoints. Changing this syntax would break all portals